### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jkrlr**
Command for sync the master of forked repo with master of main repo  has been added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,6 +109,13 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Create admin `$ python manage.py createsuperuser`
   * Run the server `$ python manage.py runserver`
 
+
+
```
Comment (garg3133) [inline README.md:113]: Please remove these extra newlines, both here and below the Pro Tip section. Keep a single newline between sections.

**Review by garg3133 — COMMENTED** [review]
Can you squash all your commits into a single commit? Also, `fix typo` is not a great commit message. Always write in your commit message exactly what you've done in the commit.

To squash your commits, follow these instructions:
* `git rebase -i HEAD~<no. of commits you made in total>`
* Except for first `pick`, replace all `pick` with `squash` in the file opened.
* Close the file after saving it.
* In the new file opened, remove all commit messages and write a new one and then close this file too after saving.
* Force push your changes. (`git push origin +<your-branch-name>`).

File: README.md (hunk 1 above)
Comment (jkrlr) [reply README.md]: okay, did it.

**Comment by jkrlr** [comment]
> Can you squash all your commits into a single commit? Also, `fix typo` is not a great commit message. Always write in your commit message exactly what you've done in the commit.
> 
> To squash your commits, follow these instructions:
> 
> * `git rebase -i HEAD~<no. of commits you made in total>`
> * Except for first `pick`, replace all `pick` with `squash` in the file opened.
> * Close the file after saving it.
> * In the new file opened, remove all commit messages and write a new one and then close this file too after saving.
> * Force push your changes. (`git push origin +<your-branch-name>`).

File: README.md [hunk 2]
```diff
@@ -107,6 +108,10 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Migrate the changes to the database `$ python manage.py migrate`
   * Create admin `$ python manage.py createsuperuser`
   * Run the server `$ python manage.py runserver`
+  
+### 💡️**Pro Tip:** 
+  * Always keep your master branch in sync with the main repository (by running `$ git pull upstream master` on your master branch) 
+  * Always create a new branch before making any changes (`$ git checkout -b <new-branch-name>`), never ever make any changes directly on the master branch.
 
  ### Setting-up the project in docker
```
Comment (jkrlr) [inline README.md:116]: @garg3133 Sir, I made changes as you mentioned.

**Comment by jkrlr** [comment]
Sir it is possible to merge this PR under GSSoC , Because I haven't open separate issue for Readme File Enhancement and their is already opened issue regarding Readme File. Since, I haven't commented on that issue also. So, I asked. 
If It's not possible and all changes made are right. Then, Please merge it.

File: README.md [hunk 3]
```diff
@@ -107,6 +108,10 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Migrate the changes to the database `$ python manage.py migrate`
   * Create admin `$ python manage.py createsuperuser`
   * Run the server `$ python manage.py runserver`
+  
+### 💡️**Pro Tip:**
```
Comment (garg3133) [inline README.md:112]: "Pro Tip" looks too big. 4 `#` should do fine. Also, try putting a space between light emoji and Pro.

**Review by garg3133 — COMMENTED** [review]
Everyhing looks good now @jkrlr expect for the three changes which I suggested below.

**Tip:** After making the changes, stage them and then run `git commit --amend` instead of the usual commit command. It will automatically update the changes in your last commit and you won't need to follow the squashing instructions again.

**Comment by garg3133** [comment]
> Sir it is possible to merge this PR under GSSoC , Because I haven't open separate issue for Readme File Enhancement and their is already opened issue regarding Readme File. Since, I haven't commented on that issue also. So, I asked.
> If It's not possible and all changes made are right. Then, Please merge it.

Labels will be made on the PRs as well so yeah, it will be counted towards your GSSoC'21.

File: README.md [hunk 4]
```diff
@@ -78,6 +78,7 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Fork the Repository.
   * Clone the repository to your local machine `$ git clone https://github.com/<your-github-username>/JagratiWebApp.git`
   * Change directory to JagratiWebApp `$ cd JagratiWebApp`
+  * Add a reference to the original repository. `$ git remote add upstream https://github.com/garg3133/JagratiWebApp.git`
```
[the remark this answers is not in the document]
Comment (jkrlr) [reply README.md:81]: > This is looking odd. Maybe we should put the who command in the new line.

Sir what does it mean *who command*. I didn't get it.

[the remark this answers is not in the document]
Comment (garg3133) [reply README.md:81]: whole* command

[the remark this answers is not in the document]
Comment (jkrlr) [reply README.md:81]: okk, got it.

File: README.md [hunk 5]
```diff
@@ -107,6 +109,10 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Migrate the changes to the database `$ python manage.py migrate`
   * Create admin `$ python manage.py createsuperuser`
   * Run the server `$ python manage.py runserver`
+  
+#### 💡️ **Pro Tip:** 
+  * Always keep your master branch in sync with the main repository (by running `$ git pull upstream master` on your local master branch) 
+  * Always create a new branch before making any changes (`$ git checkout -b <new-branch-name>`), never ever make any changes directly on the master branch.
```
Comment (jkrlr) [inline README.md:116]: Corrected the changes.

File: README.md [hunk 6]
```diff
@@ -78,6 +78,8 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Fork the Repository.
   * Clone the repository to your local machine `$ git clone https://github.com/<your-github-username>/JagratiWebApp.git`
   * Change directory to JagratiWebApp `$ cd JagratiWebApp`
+  * Add a reference to the original repository. <br>
+   `$ git remote add upstream https://github.com/garg3133/JagratiWebApp.git`
```
Comment (jkrlr) [inline README.md:82]: Putted a line break here.

**Comment by jkrlr** [comment]
@garg3133 In Readme , Clone the repository to your local machine  command also may be looks odd??

Screenshot from 2021-03-10 18-52-53

Screenshot from 2021-03-10 18-53-06

File: README.md (hunk 6 above)
Comment (garg3133) [reply README.md:82]: Don't but a line break here. Put two spaces after the sentence, it works as a line break

**Comment by garg3133** [comment]
No, clone the repo is fine.

File: README.md [hunk 7]
```diff
@@ -78,6 +78,8 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Fork the Repository.
   * Clone the repository to your local machine `$ git clone https://github.com/<your-github-username>/JagratiWebApp.git`
   * Change directory to JagratiWebApp `$ cd JagratiWebApp`
+  * Add a reference to the original repository  
+   `$ git remote add upstream https://github.com/garg3133/JagratiWebApp.git`
```
Comment (jkrlr) [inline README.md:82]: Now, Is it fine?

**Comment by jkrlr** [comment]
> No, clone the repo is fine.

oKay

File: README.md (hunk 5 above)
Comment (jkrlr) [reply README.md:116]: Now, did it.

Comment (garg3133) [reply README.md:116]: You put the `.` at the wrong place. See where I asked you to do it.

Comment (jkrlr) [reply README.md:116]: In line 114 ?

Comment (garg3133) [reply README.md:116]: Yeah

File: README.md [hunk 8]
```diff
@@ -107,6 +109,10 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Migrate the changes to the database `$ python manage.py migrate`
   * Create admin `$ python manage.py createsuperuser`
   * Run the server `$ python manage.py runserver`
+  
+#### 💡️ **Pro Tip:** 
+  * Always keep your master branch in sync with the main repository (by running `$ git pull upstream master` on your local master branch). 
+  * Always create a new branch before making any changes (`$ git checkout -b <new-branch-name>`), never ever make any changes directly on the master branch.
```
Comment (jkrlr) [inline README.md:115]: Added `.` in line 114

**Pull request merged**